Frontend/app/root/signin.model.ts

```diff
@@ -0,0 +1,6 @@
+export class SigninModel {
```

Comment:
It should. I was leaving that and a few other things for the story where we refactor a bunch of this into a user.service. Im not sure what the structure will look like after we do that so I didn't want to force a structure earlier than that. I'm imagining there will be a users directory with the registration and signin directory within it as well as the user.service.ts. So:
```
user/user.service.ts
user/registration/*
user/signin/*
```
But like i said I didnt want to force that too early or out of scope. I knew it was gonna be refactored so I left it as is in root.